ui/packages/tabs/src/TabItem.svelte

```diff
@@ -5,12 +5,14 @@
 	export let elem_id: string = "";
 	export let visible: boolean = true;
 	export let name: string;
+	export let id: string | number | object | false = {};
```

Comment:
Presumably this would no longer include "false" as an acceptable type if we make the changes above.